stablehlo/integrations/python/tests/stablehlo.py

```diff
@@ -210,8 +210,8 @@ def test_api_version():
 @run
 def test_serialization_apis():
   curr_version = stablehlo.get_current_version()
-  assert curr_version == "0.10.0"
-  
+  assert curr_version == "0.10.1"
+
```

Comment:
I wonder if we perhaps should not hardcode the version number here and instead say something along the lines of `assert curr_version is not None` or `assert type(current_version) = str`.

Reply:
```
  major_minor_patch = curr_version.split('.')
  assert len(major_minor_patch) == 3
  assert major_minor_patch[1] == "10"
```

Meaning this test would be updated on all minor version bumps. Happy to also leave that out and just test that it is in major, minor, patch format.

Reply:
"Happy to also leave that out and just test that it is in major, minor, patch format". Yeah, let's do that, perhaps via a more targeted regex like `r"\d+\.\d+\.\d+"` (mainly because I think it would be great to have a SoT for the version number).

Reply:
done